executor/linux/api.go

```diff
@@ -45,6 +48,7 @@ func (c *client) GetRepo() (*library.Repo, error) {
 }
 
 // CancelBuild cancels the current build in execution.
+// nolint: lll // process of going through steps/services/stages is verbose and could be funcitonalized
```

Comment:
I think we want to ignore the `funlen` linter instead of the `lll` linter:

```suggestion
// nolint: funlen // process of going through steps/services/stages is verbose and could be functionalized
```